src/fairchem/core/common/relaxation/ase_utils.py

```diff
@@ -33,32 +35,78 @@
 from fairchem.core.models.model_registry import model_name_to_local_file
 from fairchem.core.preprocessing import AtomsToGraphs
 
+if TYPE_CHECKING:
+    from torch_geometric.data import Batch
 
-def batch_to_atoms(batch):
+
+# system level model predictions have different shapes than expected by ASE
+ASE_PROP_RESHAPE = MappingProxyType(
+    {"stress": (-1, 3, 3), "dielectric_tensor": (-1, 3, 3)}
+)
+
+
+def batch_to_atoms(
+    batch: Batch,
+    results: dict[str, torch.Tensor] | None = None,
+    wrap_pos: bool = False,
```

Comment:
Any reason to default this to False? Feels like wrapping it back into the cell is the more correct way of doing this.